### PR TITLE
[Woo POS][Cash & Receipts] Implement UI changes

### LINKED ISSUE #13176: [Woo POS][Cash & Receipts] Implement UI changes
* Copy on the success screen
  * A card payment of...
  * A cash payment of...
* Change due color - secondary

p1734692882782219/1734615162.822509-slack-C070SJRA8DP

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #13176 

### Description

* Changes the color of the change due text on the cash payment screen
* Changes the text on the payment success screen depending on the cash/card flow

### Testing information

* Do both card and cash flows
* Notice the color of the change due text on the cash payment screen
* Notice the text on the payment success screen depending on the cash/card flow

### The tests that have been performed

Aboce

### Images/gif

https://github.com/user-attachments/assets/2c65eecb-de48-4080-81b2-9c7034c7b089

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">PR requires at least one label.</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeViewModel.kt [hunk 1]
```diff
@@ -79,7 +80,9 @@ class WooPosHomeViewModel @Inject constructor(
                 )
             }
 
-            WooPosHomeUIEvent.OnPaymentCompletedViaCash -> onOrderSuccessfullyPaid()
+            WooPosHomeUIEvent.OnPaymentCompletedViaCash -> onOrderSuccessfullyPaid(
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeViewModel.kt:83]: ❓ Looks like we are missing tests for changes in this class?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/WooPosTotalsViewModel.kt [hunk 2]
```diff
@@ -430,12 +432,16 @@ class WooPosTotalsViewModel @Inject constructor(
         }
     }
 
-    private fun showSuccessfulPaymentState() {
+    private fun showSuccessfulPaymentState(paymentMethod: PaymentMethod) {
         viewModelScope.launch {
             val dataState = dataState.value
             checkNotNull(dataState.orderTotal)
+            val template = when (paymentMethod) {
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/WooPosTotalsViewModel.kt:439]: ❓ Looks like we are missing tests for changes in this file as well?

**Review by AnirudhBhat — APPROVED** [review]
Thanks for working on this @kidinov. The functionality works as expected.

It looks like we are missing tests for changes in this PR.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeViewModel.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeViewModel.kt:83]: Added here 6534cb933f53bb9f4587cfe37d70e6d290ca6e0c

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/WooPosTotalsViewModel.kt (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/WooPosTotalsViewModel.kt:439]: Added here f124eca314214c5305c4c8e0d5f17f21c4aabb87

**Pull request merged**